scripts/import_templates_gen3.py

```diff
@@ -87,8 +87,9 @@ def _export(butler, export_file):
     runs : iterable [`str`]
         The names of the runs containing exported templates.
     """
-    skymaps = butler.registry.queryDataIds("skymap", datasets=TEMPLATE_NAME, collections=butler.collections)
-    skymap_query = f"skymap IN ({', '.join(repr(id['skymap']) for id in skymaps)})"
+    skymaps = butler.registry.queryDataIds("skymap", datasets=TEMPLATE_NAME, collections=TEMPLATE_COLLECT)
+    #skymap_query = f"skymap IN ({', '.join(repr(id['skymap']) for id in skymaps)})"
+    skymap_query = " or ".join(f"skymap = '{id['skymap']}'" for id in skymaps)
```

Comment:
Please remove the vestigial line.

Reply:
Since `IN` will never work with governor dimensions like `skymap` (IIRC that was why this change was needed?), can you also update [the script in `ap_verify_dataset_template`](https://github.com/lsst-dm/ap_verify_dataset_template/blob/main/scripts/import_templates.py#L100) with the working form?

Reply:
I've discussed this problem with @TallJimbo and we were not expecting it to trigger without any other constraints. We will file a ticket to fix the problem with `IN` such that `bind` with a list could also be used without having to form the string query yourself with `join`.